### PR TITLE
Wip lesson

### LINKED ISSUE #1: new content in progress
[Outline](https://github.com/learn-co-curriculum/sinatra-3-databases/blob/master/README.md)

deadline: 9/15/15

### PR DESCRIPTION
**Pull request by ipc103**
closes #1 - needs review @dfenjves @SophieDeBenedetto @deniznida @AnnJohn

### CODE REVIEW

**Comment by AnnJohn** [comment]
Assigned to @deniznida for review

Deadline- today, 9/14/15

File: README.md [hunk 1]
```diff
@@ -1,26 +1,255 @@
...
+
+```bash
+rake aborted!
+No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)
+/Users/iancandy/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `eval'
+/Users/iancandy/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `<main>'
+(See full trace by running task with --trace)
+```
+
+This is because we don't have a Rakefile. Let's fix this error by creating one. Create a file called `Rakefile` in the root of this directory and add the following code snippets:
```
Comment (deniznida) [inline README.md]: Should we say that no extension is needed when creating the Rakefile ?

**Comment by deniznida** [comment]
@ipc103 great job. one inline comment and fixes typos.

**Comment by AnnJohn** [comment]
@ipc103 how long would it take a student to complete this lesson?

**Comment by ipc103** [comment]
~35 - 45 min

**Comment by AnnJohn** [comment]
:+1:

**Pull request merged**